WordPress/build.gradle

```diff
@@ -18,6 +18,7 @@ repositories {
             includeGroup "org.wordpress"
             includeGroup "org.wordpress.aztec"
             includeGroup "org.wordpress.fluxc"
+            includeGroup "org.wordpress.wellsql"
```

Comment:
Shouldn't we also remove it from [the filter used on the `jcenter-mirror` in the root's `build.gradle`](https://github.com/wordpress-mobile/WordPress-Android/blob/trunk/build.gradle#L97-L98) now that it has moved to S3?

Or are we still using `1.6.0` for some of our dependencies and not moved everything to `wellsql 1.7.0` just yet (while this move to S3 is only for "> 1.7.0 version of the library" as you mention in your description) hence why keeping the two for now?

Reply:
FluxC version for the project hasn't been updated yet. Even if we updated it to the latest version in `trunk`, not all FluxC branches will be up to date to use `1.7.0` version and delaying the removal of those records removes the necessity to communicate the situation with developers.

Reply:
Ah, got it, makes sense! ✅